src/components/navBar/navBar.module.scss

```diff
@@ -0,0 +1,30 @@
+.navBar{
+  width: 100vw;
+  font-size: 16px;
+}
+
+.navBar ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: #1d1283;
+}
+
+.navBar li {
+  float: left;
+}
+
+.navBar li a {
+  display: block;
+  border-radius: 0.5rem;
```

Comment:
We don't want different `border-radius` for separate screen sizes?
Can we make it pixels?